tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -225,17 +250,24 @@ public function testEnrollAgent() {
       $validated = 0;
       foreach ($mqttACLs as $acl) {
          if (preg_match("~/agent/$serial/Command/#$~", $acl->getField('topic')) == 1) {
-            $this->integer((int) $acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_READ);
-            $validated++;
-         } else if (preg_match("~/agent/$serial/Status/#$~", $acl->getField('topic')) == 1) {
-            $this->integer((int) $acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_WRITE);
-            $validated++;
-         } else if (preg_match("~^/FlyvemdmManifest/#$~", $acl->getField('topic')) == 1) {
-            $this->integer((int) $acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_READ);
-            $validated++;
-         } else if (preg_match("~/agent/$serial/FlyvemdmManifest/#$~", $acl->getField('topic')) == 1) {
-            $this->integer((int) $acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_WRITE);
+            $this->integer((int)$acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_READ);
             $validated++;
+         } else {
+            if (preg_match("~/agent/$serial/Status/#$~", $acl->getField('topic')) == 1) {
+               $this->integer((int)$acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_WRITE);
+               $validated++;
+            } else {
+               if (preg_match("~^/FlyvemdmManifest/#$~", $acl->getField('topic')) == 1) {
+                  $this->integer((int)$acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_READ);
+                  $validated++;
+               } else {
+                  if (preg_match("~/agent/$serial/FlyvemdmManifest/#$~",
+                        $acl->getField('topic')) == 1) {
+                     $this->integer((int)$acl->getField('access_level'))->isEqualTo(\PluginFlyvemdmMqttacl::MQTTACL_WRITE);
+                     $validated++;
+                  }
+               }
+            }
```

Comment:
I disagree withthis change. nested if() is harder to understand (increases complexity)